src/cli.ts

```diff
@@ -0,0 +1,95 @@
+import * as inquirer from 'inquirer';
+
+let userInput = {
+  pluginName: '',
+  pluginType: '',
+  framework: '',
+  language: '',
+  style: ''
+}
+
+async function collectUserInput() {
+  let response;
+
+  let askPluginName = {
+    name: 'pluginName',
+    type: 'input',
+    message: 'Enter your plugin name:',
+    validate: function (value) {
+      if (value.length) {
+        return true;
+      } else {
+        return 'Please provide at least 1 character as a name.';
+      }
+    }
+  }
+  console.log('-----------------------------');
```

Comment:
Suggest to make function for a separator. It avoids multiple changes if we change separator's text.

Reply:
good idea.